### PR TITLE
Api for Performance Controller

### PR DESCRIPTION
**Pull request by ror-uday**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -33,6 +33,14 @@
     get '/emp-attendance/:id', to: 'attendences#show_attendence', as: 'show_attendance'
     get '/search', to: 'attendences#search'
   end
+
+  namespace :api do
+    namespace :v1 do
+      resources :performances
```
Comment (rorshaan) [inline config/routes.rb:39]: Why you are using all routes if you have only index action

File: spec/v1/performance_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  let(:performance) { FactoryBot.create(:performance, employee_id: employee.id) }
+
+  before do
+    sign_in(employee)
+  end
+
+  describe '#index' do
+    it 'show all performance ' do
+      get :index
+      expect(response.status).to eq(200)
```
Comment (rorshaan) [inline spec/v1/performance_spec.rb:18]: Expect with objects

File: app/controllers/api/v1/performances_controller.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+module Api
+  module V1
+    class PerformancesController < ApplicationController
+      # before_action :authenticate_employee!
+
+      skip_before_action :verify_authenticity_token
+
+      def index
+        data = Performance.all
```
Comment (rorshaan) [inline app/controllers/api/v1/performances_controller.rb:11]: use proper name of variables

**Review by rorshaan — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 1 above)
Comment (ror-uday) [reply config/routes.rb]: changes done

**Pull request closed without merging**